src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+/**
+ * This class deals with user's input by parsing them and convert each instruction
+ * into different commands.
+ */
+public class Parser {
+    private enum KeyWord {
+        END("bye"),
+        DELETE("delete"),
+        LIST("list"),
+        GET("get"),
+        SEARCH("find"),
+        DONE("done");
+
+        private final String k;
+
+        KeyWord(String k) {
+            this.k = k;
+        }
+
+        public String getK() {
```

Comment:
Perhaps the naming could be more descriptive than K? I think it would increase readability

Reply:
Would change that!